source/Agents/RandomAgent.hpp

```diff
@@ -14,72 +15,96 @@
 
 namespace walle {
 
-/**
- * Class for the Random Agent
- */
-class RandomAgent : public cse491::AgentBase {
-
-	private:
-
-        double random = 4.0;
-
-        bool moving = true;
+    /**
+     * Class for the Random Agent
+     */
+    class RandomAgent : public cse491::AgentBase {
 
-	public:
-		RandomAgent(size_t id, const std::string &name) : AgentBase(id, name) {
-		}
+        private:
 
-		~RandomAgent() = default;
+            double random = 4.0;
 
-		/// @brief This agent needs a specific set of actions to function.
-		/// @return Success.
-		bool Initialize() override {
-			return HasAction("up") && HasAction("down") && HasAction("left") && HasAction("right");
-		}
+            bool moving = true;
```

Comment:
A brief description (one line) would be good for completeness, or anything that we have defined ourselves.